src/css/default.css

```diff
@@ -34,7 +37,7 @@ body.light {
   --textLinkColor: #287cb4;
   --textLinkHoverColor: #206592;
 }
-body.dark {
+body[data-theme="dark"] {
```

Comment:
Working with a data attribute is easier than having to `document.body.classList.replace(..., ...)`